test/e2e/scenarios/namespaceselector.go

```diff
@@ -0,0 +1,14 @@
+package scenarios
+
+import (
+	. "github.com/onsi/ginkgo"
+)
+
+var _ = PDescribe("Namespace Selector based Network Policies for ingress between clusters", func() {
+	PIt("Should allow communication between pods in selected namespace", func() {
+		By("creating pod1 with lable1 in cluster1 in namespace1")
+		By("creating pod2 with lable2 in cluster2 in namespace1")
+		By("creating network policy with namespace selector")
```

Comment:
How is this test case different to the previous one?

Reply:
This just involves 2 clusters whereas the previous one involves more than 2 clusters. Network policy in both cases could be same or different.